### ISSUE TITLE
Deprecated import module SymPy seems to crash Spyder

### ISSUE THREAD

**Issue by mjgatley** [spyder-ide/spyder#13937]
### What steps will reproduce the problem?

Upon startup my variable explorer is full of 'deprecatedimportmodule's and it crashes when I try to click on them and it can no longer compute simple functions such as adding two numers together.

I am new to this so please help.

## Versions

* Spyder version: 4.1.5 
* Python version: 3.8.3
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.6.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.18.1 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
nbconvert >=4.0                :  6.0.7 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
paramiko >=2.4.0               :  2.7.2 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.2 (OK)
pygments >=2.0                 :  2.7.1 (OK)
pylint >=1.0                   :  2.6.0 (OK)
pyls >=0.34.0;<1.0.0           :  0.35.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.7 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.2.1 (OK)
spyder_kernels >=1.9.4;<1.10.0 :  1.9.4 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  19.0.2 (OK)

# Optional:
cython >=0.21                  :  0.29.21 (OK)
matplotlib >=2.0.0             :  3.3.1 (OK)
numpy >=1.7                    :  1.19.1 (OK)
pandas >=0.13.1                :  1.1.2 (OK)
scipy >=0.17.0                 :  1.5.0 (OK)
sympy >=0.7.3                  :  1.6.2 (OK)
```

**Comment by steff456** [comment]
Hi @mjgatley,

Can you please let me know how are you encountering this behavior? Also if you can upload a gif with the error will be really helpful for us to try to reproduce it

**Comment by mjgatley** [comment]
This is what my screen looks like on startup. All I have done is open spyder. I have not asked it to open any files.
[cid:bc1ceac8-febb-4720-bc33-22e726a8820f]


Then if I try to add two numbers together using F9 on line 77 the command just won't run and the red square just stays there in the console



[cid:08d9ae6c-c790-4e81-9eb7-ce917e886cf2]


If I stop it by pressing the red square, it won't run anything.

I then try to restart the kernel and have the same issue

[cid:411a1370-d296-47fe-98da-274fab64494e]



________________________________

**Comment by ccordoba12** [comment]
Hey @mjgatley, you need to upload images directly on Github. You can't send them via email.

**Comment by mjgatley** [comment]
So I completely uninstalled every piece of python software and all of sympy in order to try and fix the problem. This is the first piece of code I have tried to run in spyder. Adding two numbers at the end was just to see if the program had crashed(which it apparently has)

![image](https://user-images.githubusercontent.com/72462697/95882300-6f050e80-0d71-11eb-9943-f2169ebba9a7.png)


Please help me sort this out, I need to be able to use sympy for my university course, 
Thanks.

**Comment by ccordoba12** [comment]
Hopefully this is fixed for you in our latest version (5.4.2).
